### PR TITLE
Fix location of testutil pkg after prometheus update.

### PR DESCRIPTION
**Pull request by johncming**
* bugfix after prometheus update. https://github.com/prometheus/prometheus/pull/5893

## Changes

prometheus tsdb/testutil has moved to util/testutil, so change it in thanos too

## Verification

1. run go vet first
2. run go test ./... -v

Signed-off-by: johncming <johncming@yahoo.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
Hm, I think if we want to fix this, we need to update deps as well, right? (:

**Comment by johncming** [comment]
> Hm, I think if we want to fix this, we need to update deps as well, right? (:

Does it mean go mod？

File: go.mod [hunk 1]
```diff
@@ -33,20 +47,27 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v1.1.0
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
 	github.com/prometheus/common v0.6.0
-	github.com/prometheus/prometheus v0.0.0-20190814090039-b5c833ca2194
```
Comment (jojohappy) [inline go.mod]: Em... Why does prometheus rollback to 2.5.0? ):

File: go.mod [hunk 2]
```diff
@@ -33,20 +47,27 @@ require (
...
+	github.com/prometheus/prometheus v2.5.0+incompatible
```
Comment (jojohappy) [inline go.mod]: Em... Why does prometheus rollback to 2.5.0? ):

Comment (johncming) [reply go.mod]: > Em... Why does prometheus rollback to 2.5.0? ):

No special version is specified. According  minimal version selection algorithm, 2.5 was selected. 

Considering the updated version, there may be other effects, please turn off this pr for the time being.

thanks.

https://github.com/golang/go/wiki/Modules#version-selection

**Pull request closed without merging**